contributing/community/review-comments.rst

```diff
@@ -0,0 +1,177 @@
+
+    * ``What do you think of "RequestFactory" instead of RequestCreator?``
+
+Even if something *is* really wrong or "a bad idea", stay respectful and
+don't get into endless you-are-wrong discussions or "flame wars".
+
+Don't use hyperbole ("always", "never", "endlessly", "nothing", "worst", "horrible", "terrible").
+
+  **Don't:** ``I don't like how you wrote this code`` - there is no clear explanation why you
+  don't like how it's written.
+
+  **Better:** ``I find it hard to read this code as there many nested if statements, can you make it more
+  readable? by encapsulating some of it's details or maybe adding some comments to explain the overall logic.`` -
+  You explain why you find the code hard to read *and* give some suggestions for improvement.
+
+If a piece of code is in fact wrong, explain why:
+
+    * ``This code doesn't comply with Symfony's CS rules. Please see [...] for details``.
+
+    * ``Symfony 3 still uses PHP 5 and doesn't allow the usage scalar type-hints.``.
+
+    * ``I think the code is less readable now`` - careful here, be sure explain why you think
+      the code is less readable, and maybe give some suggestions?
+
+**Examples of valid reasons to reject:**
+
+    * We tried that in the past (link to the relevant PR) but we needed to revert it for XXX reason.
+
+    * That change would introduce too many merge conflicts when merging up Symfony branches.
+      In the past we've always rejected changes like this.
+
+    * I profiled this change and it hurts performance significantly (if you don't profile, it's an opinion, so we can ignore)
+
+    * Code doesn't match Symfony's CS rules (e.g. ``use array()`` instead of ``[]``)
+
+    * We only provide integration with very popular projects (e.g. we integrate Bootstrap but not your own CSS framework
+
+    * This would require adding lots of code and making lots of changes for a feature that doesn't look so important.
+      That could hurt maintaining in the future.
+
+Asking for changes
+------------------
+
+Rarely something is perfect from the start, while the code itself is good.
+It may not be optimal or conform the Symfony coding style.
+
+Again, understand the author already spent time on the issue and asking
+for (small) changes may be misinterpreted or seen as a personal attack.
+
+Be thankful for their work (so far), stay positive and really help them
+to make the contribution a great one. *Especially if they are a first
+time contributor.*
+
+Use words like "Please", "Thank you" and "Could you" instead of making demands;
+
+* "Thank you for your work so far. I left some suggestions for improvement
+  to make the code more readable."
+
+* "Your code contains some coding-style problems, can you fix these before
+  we merge? Thank you"
+
+* "Please use 4 spaces instead of tabs", "This needs be on the previous line";
+
+  During a pull request review you can usually leave more then one comment,
```

Comment:
Is it intended to have a leading space without `*` ?

Reply:
It's a clarification. But I agree it looks a bit strange 👍